### ISSUE TITLE
How did you split VG VQA JA ?

### ISSUE THREAD

**Issue by zengyan-97** [zmykevin/UC2#3]
Hi, 
Thanks for your interesting work! 

May I ask how did you split VG VQA JA? 
The paper says that you use the train/test split in the original VG VQA dataset. However, I didn't find the information in: https://visualgenome.org/static/data/dataset/question_answers.json.zip
Could you please refer me to the original split? 

Besides, you consider top-3129 frequent answers for VQA v2.0 and top-3000 for VG VQA JA. Is the frequency calculated on the train set or the train and validation set? 

The evaluation settings for multilingual multimodal processing is not so clear now. Hope we can make it better together. 
Thanks!

**Comment by zmykevin** [comment]
Hello Zeng,
Thank you for your interest in our work. 
The top answers are gained from only the training set. 
When I split the VG VQA dataset, I am checking if the image_data.json file from: [https://visualgenome.org/api/v0/api_home.html](https://visualgenome.org/api/v0/api_home.html)
If the file belongs to VG_100k_2, it belongs to validation; Otherwise it belongs to training. 
Hope this answers your question.

**Comment by zengyan-97** [comment]
Hi, 
As you said, I got 61534 samples for the train set 37674 for validation from a total number of 99208 samples in VG VQA JA. Is it correct? Your paper says that you split the data into 61K for training and only 30K for validation.
